WordPress/Classes/Utility/Media/FaviconService.swift

```diff
@@ -0,0 +1,107 @@
+import UIKit
+
+// Fetches URLs for favicons for sites.
+actor FaviconService {
```

Comment:
It is an experimental implementation to show site icons in "Discover" (icons not presented in the API) and for RSS subscriptions (never supported site icons). For "Discover", we'll might need to revisit this and check with the Loop team if there are any other options. For RSS, we'll still need this, but it might be worth integrating something like https://github.com/will-lumley/FaviconFinder instead.